scripts/src/webpack/webpack-resources.js

```diff
@@ -18,17 +10,17 @@ const tsOverlayConfig = tsOverlay({
   loaderOptions: {
     transpileOnly: true,
     compilerOptions: {
-      "declaration": false,
-      "declarationMap": false
-    }
+      declaration: false,
+      declarationMap: false,
+    },
```

Comment:
It seems prettier did some nontrivial changes here. I'm hoping it doesn't break anything as. web pack is only used for our web-tester iirc, which is unused.

Reply:
I think it just removed the quote marks because declaration and declarationMap are one word, so you don't need quotes for those for property names in objects.